spec/bump/commands/preview_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "spec_helper"
+
+describe Bump::CLI::Commands::Preview do
+  it 'calls the server and display the link on success' do
+    stub_and_return('previews/post_success.http')
+
+    expect do
+      new_command.call(file: __FILE__, format: 'yaml')
+    end.to output(/Preview created/).to_stdout
+
+    expect(WebMock).to have_requested(:post,'https://bump.sh/api/v1/previews')
```

Comment:
Don't you want to expect the request body is correct too? (the format and the file)

Reply:
Right !